Source/RomanNumeral.h

```diff
@@ -12,6 +12,29 @@
  * @param input The number to convert.
  * @param output String buffer in which to store the converted Roman numeral.
  */
+
+typedef struct {
```

Comment:
In Eclipse you should be able to hit `cmd + shift + F` to run the formatter. In VS Code I believe it's `cmd + shift + I`.